hamilton/function_modifiers/macros.py

```diff
@@ -1055,3 +1174,345 @@ def chain_transforms(
             )
             first_arg = raw_node.name
     return nodes, first_arg
+
+
+@dataclasses.dataclass
+class RemoteApplicableBuilder:
```

Comment:
SO my guess is that `Applicable` can subclass this... Or `Applicable` can have two modes (with/without the function). A lot of code to duplicate...